web/autograder_service.go

```diff
@@ -489,13 +489,10 @@ func (s *AutograderService) ApproveSubmission(ctx context.Context, in *pb.Approv
 
 // RefreshSubmission rebuilds the latest student submission for the logged in user
```

Comment:
Maybe the doc comment needs to be updated to say rebuild for the user specified in ...?

Reply:
Done.